app/upgrades.go

```diff
@@ -1,3 +1,138 @@
 package app
 
-func (app App) RegisterUpgradeHandlers() {}
+import (
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	bankkeeper "github.com/cosmos/cosmos-sdk/x/bank/keeper"
+	distrkeeper "github.com/cosmos/cosmos-sdk/x/distribution/keeper"
+	distrtypes "github.com/cosmos/cosmos-sdk/x/distribution/types"
+	minttypes "github.com/cosmos/cosmos-sdk/x/mint/types"
+	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
+
+	communitytypes "github.com/kava-labs/kava/x/community/types"
+	kavamintkeeper "github.com/kava-labs/kava/x/kavamint/keeper"
+	kavaminttypes "github.com/kava-labs/kava/x/kavamint/types"
+)
+
+const (
+	MainnetUpgradeName = "v0.20.0"
+	TestnetUpgradeName = "TESTNET-v0.20.0"
```

Comment:
reminder that whatever name for the upgrade we pick we have to be consistent about using a matching git tag for the code used to build the binary (cosmosvisor only cares about the name in the upgrade but our infrastracture is tag sensitive)

Reply:
ah, then maybe more consistent to name the upgrade `v0.20.0-testnet`